issue_release_status.go

```diff
@@ -31,12 +54,85 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 
 	ctx, logger := githubapp.PreparePRContext(ctx, installationID, repository, prNum)
-
-	logger.Debug().Msgf("Event action is %s", event.GetAction())
 	if event.GetAction() != "created" {
 		return nil
 	}
 
+	prBase := event.GetPullRequest().GetBase().GetRef() // The branch which the pull request is ending.
+
+	// Retrieve auto-release-policy
+
+	serviceName := event.GetRepo().GetName()
+	servicePath := handler.releaseManagerURL + "/policies?service="
+
+	// Create client
+	client := &http.Client{}
+
+	// Set route
+	resp, err := client.Get(servicePath + serviceName)
+	if err != nil {
+		return errors.Wrap(err, "issueing a GET to specified URL")
+	}
+
+	req, err := http.NewRequest("GET", servicePath+serviceName, nil) // dublicate work??
+	if err != nil {
+		return errors.Wrap(err, "wrapping new request")
+	}
+
+	req.Header.Add("Authorization", "Bearer "+handler.releaseManagerAuthToken)
+
+	resp, err = client.Do(req)
+
+	var policyResponse ListPoliciesResponse
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	defer resp.Body.Close()
+
+	json.Unmarshal(body, &policyResponse)
+
+	var autoReleaseEnvironments []string
+	var botMessage string
+
+	// Not that easy to read, sry
+	if len(policyResponse.AutoReleases) > 0 {
+		for i := 0; i < len(policyResponse.AutoReleases); i++ {
+			if policyResponse.AutoReleases[i].Branch == prBase {
+				autoReleaseEnvironments = append(autoReleaseEnvironments, policyResponse.AutoReleases[i].Environment)
+			}
+		}
+
+		botMessage += "Auto-Release Policy detected for service " + serviceName + " at branch " + prBase + "\n\n"
```

Comment:
You might see a case where there are auto release policies for other branches and thus the `autoReleaseEnvironments` is actually empty here.

Say I create a PR against branch `dev` but there are auto release policies for `master`. This will result in `policyResponse.AutoReleases` having items, but non of them match the base branch :) 

TL;DR: we should have a check again about policies on the base branch.